src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEF_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int CAPACITY_INCREASE_FACTOR = 2;
+    private Node<K, V>[] table;
+    private int size;
+    private float threshold;
```

Comment:
We do not need it as a field because it can be calculated from other fields. Including it as a field can make the code more error-prone if it is updated with an incorrect value

Reply:
comment not fixed, please remove additional variable and use other values it based on where required